deploy/.env.example

```diff
@@ -39,7 +39,6 @@ HOME_AMB_BRIDGE=
 FOREIGN_AMB_BRIDGE=
 
 # Request gas limit to be used when passing a message to the other side of the bridge
-HOME_MEDIATOR_REQUEST_GAS_LIMIT=
```

Comment:
Does it make sense to have this limit be applied by default on the home side of the bridge?

Reply:
Done. Added a module constructor argument with the default request gas limit which is taken from this env variable.